css/style.css

```diff
@@ -0,0 +1,107 @@
+:root {
+    --main-white: #ffff;
+    --main-green: #00FF00;
+    --main-grey: #808080;
+    --main-black: #000000;
+
+}
+
+
+@media (max-width: 30em){
+    html {
+        font-size : 50%
+    }
+}
+
+#welcome-section {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items:center;
+    width: 100%;
+}
+
+#navbar {
+display: flex;
+justify-content: flex-end;
+position: fixed;
+top:0;
+left:0;
+width: 100%;
+background: rgb(255,255,255);
+background: radial-gradient(circle, rgba(255,255,255,1) 0%, rgba(70,252,152,1) 100%);
+box-shadow: 0 2px 0 rgba(27, 26, 26, 0.4);
+}
```

Comment:
Each line should have 2 empty spaces at the beginning.
Your code doesn't seem very organized.
Also, make sure there is one empty line after each styling you make. 🎨